Assets/Dependencies/SmashBrew/Character/States/MovementState.cs

```diff
@@ -142,7 +142,7 @@ public enum CharacterFacingMode {
             get { return _ledgeTarget; }
         }
 
-        public Transform CurrentLedge {
+        public GameObject CurrentLedge {
             get { return _currentLedge; }
             set {
                 bool grabbed = _currentLedge == null && value != null;
```

Comment:
Er... are you sure? Isn't this method called on clients to snap players to the ledge?